### DISCUSSION TITLE
喜欢做家务是优点吗？ | /posts/jiawu/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#608] [Announcements]
https://yuanfan.rbind.io/posts/jiawu/

**Comment by liechi** [comment]
完全不明白你这几个同事要表达什么意思。

喜欢做饭做家务对男女来说都是优点。不喜欢做饭做家务也是男女都有的权利---只要双方有共识，每天相约下馆子也很好。

除了怀孕男性无法完成外，生活中似乎没有什么分工应以性别为界。

**Reply by earfanfan** [reply]
我也没明白。如果下次再跟小亮一起打球的话，我可以把这篇博客拿去给他看看，问问他当时具体是要表达什么意思。不过既然当我提到“为什么婚姻中都让女生奉献，不让男生奉献，大家都奉献”时，他的反馈并不是否定原先所要表达不是只让女生奉献，而是说我“你们女生上纲上线”，我猜他可能有点怀疑我只是想打拳，不一定能够得到答案。

另外，博客中提到的三个同事的思想观念应该并不完全一致。对于小全这个样本，只分析他的言论的话，他是把女性的他者奉献和取悦男性混为一谈的。而我最近看《被讨厌的勇气》里面提到的他者贡献是指，在充分自我接纳、他者信赖的基础上继续追求他者贡献，不分性别。其实你说的是在社会中男女达到平等的理想情况呀，我接触到的世界的这一面，仍然以性别年龄等划定了很多分界线。

**Reply by earfanfan** [reply]
@liechi 

喂，列弛，你的博客好久没耕了啊。春天到了，万物复苏，快去赏赏花，看看风景，吃吃美食，然后回去耕博客吧。

**Reply by liechi** [reply]
是啊，田园将芜胡不归？
以往非不谏，来者亦可追。

**Reply by earfanfan** [reply]
陶渊明说：列弛呀，你真是个好孩子，旗帜立得如此鲜明，那么就慢慢悠悠去耕吧，俺等着看哩。

**Reply by hongtaoh** [reply]
小凡在催每个人更新博客。

**Reply by earfanfan** [reply]
不啊，没有催每个人，就是去列弛的园子瞅了瞅，发现又荒了一年多，于是催耕一下，然后又顺手去益辉那催耕了一下。

**Comment by hongtaoh** [comment]
我想了一会儿才知道小凡是你。为什么我觉得现在男生做家务比较多？可能我出来久了，又都是一个人，就觉得男生做家务是天经地义的事情。

**Reply by earfanfan** [reply]
这个世界很大，咱们都只能接触到一小部分，要是用身边统计学的思维按少量样本来归纳结论就会这样。一个人独立生存，自己做家务确实就是天经地义的事，这个不分性别的说。但若是组建家庭共同生活的话，我也是认同列弛的观点，不该把家务都丢给女性或者都丢给男性，而是要达成共识如何分配。

哈哈哈，其实把自己经历的事情尽量用第三人称视角来写，也蛮有意思的，这样应该也不会误导别人只带入“我”的视角。或许以后你也可以试试写“小涛”的对话故事。
